src/Common/src/Interop/Windows/kernel32/Interop.GetComputerName.cs

```diff
@@ -11,6 +12,9 @@ internal partial class Kernel32
         [DllImport(Libraries.Kernel32, CharSet = CharSet.Unicode, EntryPoint = "GetComputerNameW")]
         private static extern unsafe int GetComputerName(char* lpBuffer, ref uint nSize);
 
+        [DllImport(Libraries.Kernel32, CharSet = CharSet.Unicode, BestFitMapping = false)]
+        public static extern bool GetComputerName(StringBuilder lpBuffer, int[] nSize);
```

Comment:
If we expect this to work with .NET Native I don't think we can interop with StringBuilder (at least that used to be the case).

Reply:
We don't currently expect it to work with .NET Native at least I have not asked for the API to be allowed and much of perf counters requires elevation, let alone app container.